src/test_pixel_gap_cuts.cpp

```diff
@@ -72,6 +74,12 @@ int main(int argc, char** argv) {
 
         auto cellID = sim_hit.getCellID();
         auto pos    = sim_hit.getPosition();
+        auto mom    = sim_hit.getMomentum();
+        TVector3 p; p.SetX(mom.x); p.SetY(mom.y); p.SetZ(mom.z);     
```

Comment:
```suggestion
        TVector3 p(mom.x, mom.y, mom.z);
```

Reply:
The one you suggested was what I tried for the first time. The compilation failed and it complained that this method not found (LoL) and so I had to move to SetX(). This morning I have updated my `eic-shell`.